vms/rpcchainvm/vm_server.go

```diff
@@ -333,56 +333,24 @@ func (vm *VMServer) Shutdown(ctx context.Context, _ *emptypb.Empty) (*emptypb.Em
 	return &emptypb.Empty{}, errs.Err
 }
 
-func (vm *VMServer) CreateHandlers(ctx context.Context, _ *emptypb.Empty) (*vmpb.CreateHandlersResponse, error) {
-	handlers, err := vm.vm.CreateHandlers(ctx)
+func (vm *VMServer) NewHTTPHandler(ctx context.Context, _ *emptypb.Empty) (*vmpb.NewHTTPHandlerResponse, error) {
+	handler, err := vm.vm.NewHTTPHandler(ctx)
```

Comment:
Do we require all VMs to return an http.Handler now? We used to support not specifying any.

Reply:
fixed + added a regression test